proto/hummock.proto

```diff
@@ -247,9 +247,10 @@ message CompactTask {
     SEND_FAIL_CANCELED = 6;
     MANUAL_CANCELED = 7;
     INVALID_GROUP_CANCELED = 8;
-    EXECUTE_FAILED = 9;
```

Comment:
Could this break backwards compatibility? I prefer not to modify the serial number of proto

Reply:
> Could this break backwards compatibility? I prefer not to modify the serial number of proto

Currently it's safe because we do not distinguish these errors when handling compaction error.